src/python/devcontainer-feature.json

```
@@ -1,6 +1,6 @@
 {
   "id": "python",
-  "version": "1.0.22",
+  "version": "1.0.23",
```

```suggestion
  "version": "1.1.0",
```

As this PR adds a new Feature option, can we bump the minor version? thanks!